src/libs/ReportUtils.ts

```diff
@@ -4618,6 +4625,61 @@ function shouldDisableThread(reportAction: OnyxEntry<ReportAction>, reportID: st
     );
 }
 
+function getAllAncestorReportActions(
+    report: Report | null | undefined,
+    shouldHideThreadDividerLine: boolean,
+    reports: OnyxCollection<Report> = {},
+    reportActions: OnyxCollection<ReportActions> = {},
+): Ancestor[] {
+    if (!report) {
+        return [];
+    }
+    const convertReports: OnyxCollection<Report> = {};
+    const convertReportActions: OnyxCollection<ReportActions> = {};
+    Object.values(reports ?? {}).forEach((itemReport) => {
+        if (!itemReport) {
+            return;
+        }
+        convertReports[itemReport.reportID] = itemReport;
+    });
+    Object.keys(reportActions ?? {}).forEach((actionKey) => {
+        if (!actionKey) {
+            return;
+        }
+        const reportID = CollectionUtils.extractCollectionItemID(actionKey as `reportActions_${string}`);
+        convertReportActions[reportID] = reportActions?.[actionKey] ?? null;
+    });
+    const allAncestors: Ancestor[] = [];
+    let parentReportID = report.parentReportID;
+    let parentReportActionID = report.parentReportActionID;
+    // Store the child of parent report
+    let currentReport = report;
+    let currentUnread = shouldHideThreadDividerLine;
+    while (parentReportID) {
+        const parentReport = convertReports?.[parentReportID];
+        const parentReportAction = convertReportActions?.[parentReportID]?.[parentReportActionID ?? ''] ?? null;
+        if (!parentReportAction || ReportActionsUtils.isTransactionThread(parentReportAction) || !parentReport) {
```

Comment:
Why this `ReportActionsUtils.isTransactionThread(parentReportAction)`?

Reply:
If we never want to show the ancestry then I would do this check sooner.

Reply:
@dukenv0307 Could you answer this?

Reply:
I think this is resolved by my comment above.